packages/mui-base/src/utils/clamp.ts

```diff
@@ -0,0 +1 @@
+export { default as clamp } from '@mui/utils/clamp';
```

Comment:
Can we do the utils changes in a separate PR? It will make the review easier.

Reply:
I think at this stage, it's easier if the utils are kept in this PR. To keep it working before rebasing, I'd have to import directly from `@mui/utils` temporarily, right? I can remember to do utils separately for next time.